pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/OncHistoryTestUtil.java

```diff
@@ -445,4 +451,24 @@ public void assertSmIdIsNOTDeleted(String participantGuid,  int tissueId, int sm
         }
 
     }
+
+    public void deleteOncHistoryDirectlyFromDB(int oncHistoryDetailId){
```

Comment:
Maybe move this to a dao?  Seems like it could be a useful thing outside of testing.